com.unity.netcode.gameobjects/Runtime/Core/NetworkManager.cs

```diff
@@ -240,7 +242,7 @@ public ulong LocalClientId
         [HideInInspector] public NetworkConfig NetworkConfig;
 
         /// <summary>
-        /// The current hostname we are connected to, used to validate certificate
+        /// The current host name we are connected to, used to validate certificate
```

Comment:
@TwoTenPvP since we don't do authentication/encryption anymore this connected host name thingy shouldn't exist anymore right?

Reply:
If it is, could we leave that for a different PR?  I was simply fixing a spelling/typo mistake.  :)

Reply:
we could leave this whole change to a different PR entirely :)

Reply:
we could leave this change entirely for later :)